### PR TITLE
Corp Mercs Style SL Pointer/Locator 

### PR DESCRIPTION
**Pull request by spookydonut**
:cl: Surrealistik
-Implements a responsive Corp Mercs style SL pointer/arrow that is highly functional unlike the near-worthless current one.
/:cl:

### CODE REVIEW

**Comment by spookydonut** [comment]
done

File: code/controllers/stonedmc/subsystem/direction.dm [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+	for(var/a in leader_mapping)
+		processing_mobs.Add(a)
+		processing_mobs[a] = list()
+	return ..()
+
+/datum/controller/subsystem/direction/stat_entry()
+	var/mobcount = 0
+	for(var/L in processing_mobs)
+		mobcount += length(processing_mobs[L])
+	..("P:[mobcount]")
```
Comment (Rohesie) [inline code/controllers/stonedmc/subsystem/direction.dm]:
```suggestion
	return ..("P:[mobcount]")
```

File: code/modules/clothing/head/helmet.dm [hunk 2]
```diff
@@ -161,12 +161,13 @@
...
 /obj/item/clothing/head/helmet/marine
 	name = "\improper M10 pattern marine helmet"
-	desc = "A standard M10 Pattern Helmet. It reads on the label, 'The difference between an open-casket and closed-casket funeral. Wear on head for best results.'."
+	desc = "A standard M10 Pattern Helmet. It reads on the label, 'The difference between an open-casket and closed-casket funeral. Wear on head for best results.'. Contains a small built-in camera."
 	icon = 'icons/obj/clothing/cm_hats.dmi'
 	sprite_sheet_id = 1
 	icon_state = "helmet"
 	armor = list(melee = 65, bullet = 35, laser = 30, energy = 20, bomb = 25, bio = 0, rad = 0)
 	health = 5
+	var/obj/machinery/camera/camera
```
Comment (Rohesie) [inline code/modules/clothing/head/helmet.dm]: This is merge garbage, the whole file.

File: code/modules/mob/living/carbon/human/human_movement.dm [hunk 3]
```diff
@@ -82,6 +82,37 @@
...
+	. = ..()
+	Process_SL_Locator()
+*/
+
+/mob/living/carbon/human/proc/clear_leader_tracking()
+	var/obj/screen/SL_dir = hud_used.SL_locator
+	SL_dir.icon_state = "SL_locator_off"
+
+/mob/living/carbon/human/proc/update_leader_tracking(var/mob/living/carbon/human/H)
+	/*if(!sl_headset_active)
```
Comment (Rohesie) [inline code/modules/mob/living/carbon/human/human_movement.dm]: Remove commented code.

File: code/modules/mob/living/carbon/human/life/handle_regular_hud_updates.dm [hunk 4]
```diff
@@ -56,8 +56,8 @@
 			else
 				clear_fullscreen("brute")
 
-		if(hud_used.locate_leader && hud_used.locate_leader.alpha && prob(25)) //not invisible, 25% to not call it all the time
-			locate_squad_leader()
+		//if(world.time > (sl_locator_next_update + HUD_SL_LOCATOR_PROCESS_COOLDOWN))
```
Comment (Rohesie) [inline code/modules/mob/living/carbon/human/life/handle_regular_hud_updates.dm]: This one as well.

File: html/changelogs/AutoChangeLog-pr-224.yml [hunk 5]
```diff
@@ -0,0 +1,8 @@
+author: "Surrealistik"
```
Comment (Rohesie) [inline html/changelogs/AutoChangeLog-pr-224.yml]: Should this be here?

File: code/modules/clothing/head/helmet.dm (hunk 2 above)
Comment (spookydonut) [reply code/modules/clothing/head/helmet.dm]: i had to deal with addition and removal in this file and made mistakes resolving conflicts

**Pull request merged**